raiden-dapp/.env.raiden-package

```diff
@@ -0,0 +1 @@
+VUE_APP_RAIDEN_PACKAGE=http://localhost:8080/wizard.json
```

Comment:
May you please use a relative url here, like `/wizard.json`, so it'll be agnostic in regard to `hostname:port`